_posts/2020-06-01-Fortran-Newsletter-June-2020.md

```diff
@@ -0,0 +1,119 @@
+---
+layout: post
+title: "Fortran newsletter: June 2020"
+category: newsletter
+author: Jeremie Vandenplas, Brad Richardson, and Milan Curcic
```

Comment:
Milan, you should probably be the main author.

Reply:
I think the order is good as is. You originated the newsletter and Brad made a significant addition. I merely tidied things up.